examples/apache_metrics.mtail

```diff
@@ -3,11 +3,12 @@
 
 # Parser for a metrics-friendly apache log format
 # LogFormat "%v:%p %R %m %>s %H conn=%X %D %O %I %k" metrics
-counter apache_http_requests_total by server_port, handler, request_method, request_status, request_protocol
 counter apache_http_connections_total by server_port, handler, request_method, request_status, request_protocol, connection_status
-counter apache_http_requests_time_microseconds_total by server_port, handler, request_method, request_status, request_protocol
-counter apache_http_sent_bytes_total by server_port, handler, request_method, request_status, request_protocol
 counter apache_http_received_bytes_total by server_port, handler, request_method, request_status, request_protocol
+counter apache_http_request_time_microseconds by quantile, server_port, handler, request_method, request_status, request_protocol
```

Comment:
"quantile" doesn't seem like the right term here. It's the boundary of the latency bucket we are talking about here. So "bucket" or "upper_bound" or something?